### ISSUE TITLE
Makey Makey Question

### ISSUE THREAD

**Issue by patrickmoberg** [billybrown/pumpkins#3]
Brentman, I've got the unit you sent wired up.  Thanks again!  It appears from the LED on the board that it's registering a completed loop (i.e. I'm holding a green wire and tapping the others), but I don't know if my computer is getting the signal properly.  Is there any software I need to be running or something I'm missing?  I'm on OS X Sierra.

**Comment by watsonboyett** [comment]
Hmm. It *should* show up as a keyboard. Maybe try opening a text editor and
see if it prints anything? About 10 of the wires are mapped to letters,
while the rest are mapped to arrow keys and space bar.

I've tested it on Windows and it worked without installing anything extra.
I'll test it on my Mac and see what happens.

**Comment by watsonboyett** [comment]
When I first plugged the MaKey clone into my Mac, it asked me to identify the keyboard by pressing certain keys. I ended up just closing that setup panel, since those keys aren't programmed into the MaKey. I opened a text editor and touched the green ground wire to all of the other wires and it still printed the right keyboard characters (qwerty + asdfg + arrow keys).

There are three green LEDs on the board. Whenever the board is powered, the LED near the center will stay on constantly. Once the board "boots up", a second green LED (near the edge) will start flashing about two times per second. Whenever the the board detects a completed circuit between the green ground wire and any of the other wires, then the third LED will light up (it will only turn off once the wires are detached from each other). Let me know if you aren't seeing all three LEDs light up.

I've also found that some of my micro USB cables are "power only" and don't allow for any data to be sent between the computer and device. So maybe double-check that the USB cable is good too?
